package.json

```diff
@@ -47,6 +47,10 @@
     ]
   },
   "private": true,
+  "workspaces": [
+    "apps/**",
+    "libs/**"
+  ],
```

Comment:
We now use Yarn workspaces as the `docs` project uses a version of TypeScript different from the other projects in our Nx workspace.